canonical-websites/maas.io
# Updated to latest version of cloud vanilla and fixed link colours

## Done

- Changed cloud-vanilla to be the latest version
- Updated all buttons in the site to use the cloud colour scheme
- All links are now blue and resolved issues with the external versions
- Fixed various design issues once upgrading to the latest version of cloud vanilla

## QA

- Pull the branch, remove ```node_modules``` and run ```npm i```. 
- Build new CSS run ```make sass```.
- Run site ```make develop``` and check all pages for any issues.

## Issue / Card

n/a

## Screenshots

n/a
